slices/dotnet-sdk-8.0.yaml

```diff
@@ -0,0 +1,41 @@
+package: dotnet-sdk-8.0
+
+essential:
+  - dotnet-sdk-8.0_copyright
+
+slices:
+  # libicu is not needed if the environment is set to invariant by using
+  # the environment variable DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1
+  # If globalization is needed, libicu must be included as well.
+  # For more information, see:
+  # https://learn.microsoft.com/en-us/dotnet/core/runtime-config/globalization
+  # https://github.com/dotnet/runtime/blob/main/docs/design/features/globalization-invariant-mode.md
+  core:
+    essential:
+      - aspnetcore-runtime-8.0_core
+      - aspnetcore-targeting-pack-8.0_libs
+      - ca-certificates_data
+      - dotnet-apphost-pack-8.0_libs
+      - dotnet-runtime-8.0_libs
+      - dotnet-targeting-pack-8.0_libs
+      - dotnet-templates-8.0_libs
+      - libc6_libs
+      - libgcc-s1_libs
+      - libstdc++6_libs
+      - netstandard-targeting-pack-2.1-9.0_libs
+    contents:
+      /usr/lib/dotnet/metadata/workloads/8.0.100/userlocal:
+      /usr/lib/dotnet/packs/Microsoft.AspNetCore.App.Runtime.*/8.0.*/**:
+      /usr/lib/dotnet/packs/Microsoft.NETCore.App.Runtime.*/8.0.*/**:
+      /usr/lib/dotnet/sdk-manifests/8.0.100/microsoft.net.sdk.aspire/8.0.0-preview.*/**:
+      /usr/lib/dotnet/sdk-manifests/8.0.100/microsoft.net.workload.*/**:
+      /usr/lib/dotnet/sdk/8.0.*/**:
+
+  standard:
+    essential:
+      - dotnet-sdk-8.0_core
+      - libicu76_libs
```

Comment:
Probably worth a comment here about libicu as well, even if it just references the one above.